Model/Service/ReportNewRules.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Flancer32\Csp\Model\Service;
+
+use Flancer32\Csp\Api\Data\Fl32RuleSentInterfaceFactory;
+use Flancer32\Csp\Api\Fl32RuleSentRepositoryInterface;
+use Flancer32\Csp\Api\Repo\Data\Rule as ERule;
```

Comment:
what is the meaning of the `ERule` alias?

Reply:
It was copied from the original usage and not refactored

Reply:
OK